### PR TITLE
389 - added sub-rules, match most severe, and better error message when condition isn't correct. 

### PR DESCRIPTION
**Pull request by hk-dassana**

### CODE REVIEW

**Review by parth-dassana — DISMISSED** [review]
LGTM, amazing work!

**Comment by hk-dassana** [comment]
> Please add unit test cases @hk-dassana

Please see attached.
[Untitled.pdf](https://github.com/dassana-io/dassana/files/7449872/Untitled.pdf)

File: engine/src/main/java/app/dassana/core/risk/eval/RiskEvaluator.java [hunk 1]
```diff
@@ -29,35 +36,130 @@ public RiskEvaluator() {
   Scope rootScope = Scope.newEmptyScope();
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
-  public Risk evaluate(RiskEvalRequest input) {
-    Risk risk = new Risk();
+  // a string array to hold the risk severities in descending order (from critical to low)
+  private static final String[] riskValues = new String[]{ContentManager.RISKS.CRITICAL.getSeverity(),
```
[the remark this answers is not in the document]
Comment (hk-dassana) [reply engine/src/main/java/app/dassana/core/risk/eval/RiskEvaluator.java]: moved this as method in contentManager

**Review by parth-dassana — DISMISSED** [review]
LGTM

**Comment by parth-dassana** [comment]
Btw @hk-dassana some of your tests broke because of a change I made. I am not outputting the `vendorSeverity` as well.

**Pull request merged**